### ISSUE TITLE
Docker image with embeded Tomcat

### ISSUE THREAD

**Issue by paulo-graca** [DSpace/DSpace#9294]
**Is your feature request related to a problem? Please describe.**
The merge of https://github.com/DSpace/DSpace/pull/8797 introduces the embeded tomcat feature. Saying it so, I would be useful if docker images could be simplified also by using it.

**Describe the solution you'd like**
We need to refactor Dockerfile and change the existing commands to be compatible with this way of deployment. Meaning you don't need to do all of this https://github.com/DSpace/DSpace/blob/main/Dockerfile#L52-L66. Considering the instructions here: https://github.com/DSpace/DSpace/pull/8797#issuecomment-1876630660 it would be something like:
```
mvn spring-boot:run -Dspring-boot.run.arguments="--dspace.dir=/path/to/install/folder/ --logging.config=file:///path/to/install/folder/config/log4j2-console.xml"
```

**Comment by paulo-graca** [comment]
Isn't this already addressed with:
https://github.com/DSpace/DSpace/commit/6a012c8fc195eb545e8b2cc085a8d38f1429385b ?

**Comment by tdonohue** [comment]
@paulo-graca : Yes, you are correct.  This was fixed by #9505.  Closing as fixed in 8.0
